scalyr_agent/configuration.py

```diff
@@ -2330,6 +2338,13 @@ def __verify_main_config_and_apply_defaults(
         self.__verify_or_set_optional_bool(
             config, "disable_leak_bandwidth_stats", False, description, apply_defaults
         )
+        self.__verify_or_set_optional_bool(
+            config,
+            "disable_leak_copy_manager_stats",
```

Comment:
I don't think you mean `leak` here.

Reply:
I went with how the other 2 status message disable flags were defined, I have no idea why they have "leak" there actually. I'm fine removing it.

Reply:
Those were status messages put in to help look specifically for memory leak issues.